lib/voters.js

```diff
@@ -110,7 +110,7 @@ api.get = callbackify(async (
   return voter;
 });
 
-api.getLedgerNodeId = callbackify(async (voterId) => {
+api.getLedgerNodeId = callbackify(async voterId => {
```

Comment:
I guess we still need these `callbackify` calls because of the test suite? +1 for removing them as soon as we can.

Reply:
Agreed.